### PR TITLE
Do not write the fixed output twice

### PR DESCRIPTION
**Pull request by SpacePossum**

### CODE REVIEW

File: Symfony/CS/Fixer.php [hunk 1]
```diff
@@ -294,10 +299,6 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
             if ($diff) {
                 $fixInfo['diff'] = $this->stringDiff($old, $new);
             }
-
-            if (!$dryRun) {
-                file_put_contents($file->getRealpath(), $new);
-            }
```
Comment (SpacePossum) [inline Symfony/CS/Fixer.php]: I guess this is the result of merge to master somewhere?

File: Symfony/CS/FileCacheManager.php [hunk 2]
```diff
@@ -139,6 +139,8 @@ private function saveToFile()
             )
         );
 
-        file_put_contents($this->cacheFile, $data, LOCK_EX);
+        if (false === @file_put_contents($this->cacheFile, $data, LOCK_EX)) {
+            throw new IOException(sprintf('Failed to write file "%s".', $this->cacheFile), 0, null, $this->cacheFile);
```
Comment (keradus) [inline Symfony/CS/FileCacheManager.php]: no need to import ?

File: Symfony/CS/Fixer.php (hunk 1 above)
Comment (keradus) [inline Symfony/CS/Fixer.php]: Yeah, probably.
Great catch!

File: Symfony/CS/FileCacheManager.php (hunk 2 above)
Comment (SpacePossum) [inline Symfony/CS/FileCacheManager.php]: good catch, done.

**Comment by keradus** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @SpacePossum.

**Pull request merged**